packages/cli/__tests__/zosfiles/__system__/view/ds/View.ds.system.test.ts

```diff
@@ -0,0 +1,77 @@
+/*
```

Comment:
Curious why GH doesn't show the  corresponding `.snap` file (in a PR view) for these tests 🤔 

![image](https://user-images.githubusercontent.com/37381190/168156726-9a95daab-0d2c-4e9f-a202-32fbf017f006.png)